sdk/harambe/core.py

```diff
@@ -474,6 +475,10 @@ async def run(
         if isinstance(url, Path):
             url = f"file://{url.resolve()}"
 
+        async def default_goto_error_cb(url: str, status: int):
+            raise GotoError(url, status)
```

Comment:
you can move this outside by making this default at the funcion level above

Reply:
💀 I'm dumb lmao